media/js/views.js

```diff
@@ -80,15 +78,19 @@ $(document).ready( function() {
 
     // button behavior logic, see css
     function buttonLogic() {
-        $('.js-get-form, .js-create-object, .update, .cancel').addClass( 'hover' );
+        var selectors = '.js-get-form, .js-create-object, .update, ' +
+            '.cancel, .system_form';
+        $(selectors).addClass( 'hover' );
         if ( $(this).hasClass( 'selected' ) ) {
             $(this).removeClass( 'selected' );
         } else {
-            $('.js-get-form, .js-create-object, .update, .cancel').removeClass( 'selected' );
+            $(selectors).removeClass( 'selected' );
             $(this).removeClass( 'hover' ).addClass( 'selected' );
         }
     }
-    $( document ).on('click', '.js-get-form, js-create-object, .update, .cancel', buttonLogic );
+    $( document ).on('click',
+        '.js-get-form, .js-create-object, .update, .cancel, .system_form',
+        buttonLogic );
```

Comment:
I am gonna leave as is for now. I am gonna do a simple pr to add a js-buttonLogic selector.